test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -1005,18 +1005,10 @@ if Version.match?(System.version(), ">= 1.13.0") do
         """
       )
 
-      assert_formatter_output(
-        """
-        <div><pre>Text
-        Text</pre></div>
-        """,
-        """
-        <div>
-          <pre>Text
-        Text</pre>
-        </div>
-        """
-      )
+      assert_formatter_doesnt_change("""
+      <div><pre>Text
+      Text</pre></div>
```

Comment:
It feels like this one should break because there is a newline inside? What do you think about this?

```
  defp block_to_algebra(block, %{mode: :preserve} = context) do
    concat =
      Enum.reduce(block, empty(), fn node, doc ->
        {_type, next_doc} = to_algebra(node, context)
        concat(doc, next_doc)
      end)

    force_unfit? =
      Enum.any?(block, fn
        {:text, text, %{newlines: newlines}} -> newlines > 0 or String.contains?(text, "\n")
        _ -> false
      end)

    if force_unfit? do
      concat |> force_unfit() |> group()
    else
      concat |> group()
    end
  end
```